source/reference/limitations.txt

```diff
@@ -53,6 +53,9 @@ General Limitations
 - `Queryable Execution
   <https://www.mongodb.com/docs/v6.0/core/queryable-encryption/>`__ is
   not supported.
+- If the :dbcommand:`renameCollection` or the :dbcommand:`drop` command
+  modifies a :ref:`capped collection <manual-capped-collection>`,
+  synchronization is not supported.
```

Comment:
It seems there could be a couple of outcomes. The jira ticket says there may be some empty collections created (which is regarded as inconsistency) and/or mongosync could crash.

Reply:
In that case, consider adding the following clarification:

"... synchronization is not supported for the cluster containing the capped collection."

This may help users understand that it's not just the one collection that's affected, but potentially the entire cluster.